client/components/CheckoutCartSumm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import CheckoutCartSummCard from './CheckoutCartSummCard'
+
+const CheckoutCartSumm = ({items}) => {
+  //const {orderItem} = items
+
+  const numberWithCommas = num => {
+    return num.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')
+  }
```

Comment:
I think you can just do "num.toLocaleString()" and that should convert the number to having commas

Reply:
done